internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -226,8 +230,15 @@ spec:
             # remove the history as it breaks the results build up
             jq -r 'del(.state_history)' <<< \${build_info} | jq -c . > $(results.jsonBuildInfo.path)
             case \${state} in
-              "complete") break ;;
-              "failed") break ;;
+              "complete") echo -n "Build succeeded so IIB log is not included" > \$(results.iibLog.path); break ;;
```

Comment:
Even for a successful build it would be nice to pass the iib build url and print it in the release pipeline, no?

Reply:
Not sure - would anyone really care to check a log in case of success? I never look at any logs personally when things pass, that is why I did it this way

Reply:
OK, if you don't want to add it, it's fine, it's not required. But I think it's a nice thing to have - if something unexpected happens and you want to debug (as a user). And in this case it would be trivial to do. But it's true that we don't have anything like that for other pipelines here.

Reply:
I am not sure the catalog task will pass e2e until this is merged actually because I exit the catalog task with the exitCode result. So I may need to merge it in stage to get e2e to pass and just have stage always pass for a week

Reply:
I'll probably merge this once the catalog PR is approved and e2e is fixed. Then I won't promote this to prod until the catalog task change is being promoted to prod